pkgs/development/libraries/qca2/default.nix

```diff
@@ -14,6 +14,12 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  # tells CMake to use this CA bundle file if it is accessible
+  preConfigure = ''export QC_CERTSTORE_PATH=/etc/ssl/certs/ca-certificates.crt'';
```

Comment:
What happens at runtime?

Reply:
The path to `/etc/ssl/certs/ca-certificates.crt` is compiled into the library and is used at runtime.